modules/sbom-about.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-vulnerabilities/scanner-generate-sbom.adoc
+:_mod-docs-content-type: CONCEPT
+[id="sbom-about_{context}"]
+= About SBOMs
+
+A Software Bill of Materials (SBOM) is a digital record that lists the components of a piece of software and their origins. It provides sufficient depth of information to establish reliable provenance. It can provide other useful information, for example, how the software was assembled, license information, or known vulnerabilities. It should be cryptographically signed to ensure its integrity.
+
+SBOMs facilitate tracking license compliance, as well as vulnerability awareness, by giving users a more precise and ideally, exhaustive, view of their software stack. Vulnerability information can be mapped to SBOMs, and generally comes from other sources, such as manifests, Red Hat Security Advisories, or the NIST National Vulnerability Database.
+
+Organizations can use SBOMs to locate the presence of vulnerable software packages and components and respond more quickly to mitigate the risk. Additionally, being able to generate SBOMs assists organizations in complying with link:https://www.nist.gov/itl/executive-order-14028-improving-nations-cybersecurity[Executive Order 14028: Improving the Nations Cybersecurity].
```

Comment:
Although this section is true for other types of SBOMs, for "Analyzed" type SBOMs many of these benefits do not apply.  

The SBOM ACS is producing does NOT:

- establish reliable provenance
- show how the software was assembled
- have license information
- show known vulnerabilities.
- and is not cryptographically signed.

Consider removing the first few sections or perhaps consolidation with the  "Red Hat Advanced Cluster Security and SBOMs" section